### PR TITLE
Add Call Control and other call ids to call state

### PR DESCRIPTION
**Pull request by hugobdev**
The RTC backend is going to send `telnyx_call_control_id`,
`telnyx_session_id` and `telnyx_leg_id` params in the
`telnyx_rtc.invite` message. We then save it inside the Call object so
that it is accessible in upcoming `notification` events.

This should enable customers to cross-reference calls between the Call
Control backend and the WebRTC client.

This PR includes a few fixes to the VertoHandler tests.

## 📝 To Do

- [x] All linters pass
- [x] All tests pass
- [x] Change documentation based on my changes

## ✋ Manual testing

1. `npm link` in this repository
1. `npm link @telnyx/webrtc` in the repository you will run this code on (e.g. call-center/web-client)
1. Open the network inspector and inspect `rtc.telnyx.com` messages
1. Call your webrtc client
1. Inspect the incoming `telnyx_rtc.invite` message params. Confirm it has `telnyx_session_id` and `telnyx_leg_id`. `telnyx_call_control_id` to be implemented.
1. Confirm you got a notification event with `telnyxSessionId` and `telnyxLegId` as call options.

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [x] Firefox
- [ ] Safari

## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |            |
| usage.gif   |            |

### CODE REVIEW

File: packages/js/src/Modules/Verto/tests/Verto.test.ts [hunk 1]
```diff
@@ -1,275 +1,413 @@
-import behaveLikeBaseSession from './behaveLike/BaseSession.spec'
-import VertoHandler from './webrtc/VertoHandler'
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/tests/Verto.test.ts:2]: Removed this VertoHandler import

File: packages/js/src/Modules/Verto/tests/Verto.test.ts [hunk 2]
```diff
@@ -1,275 +1,413 @@
...
+  };
+  let instance: Verto = null;
 
-  const noop = (): void => { }
+  const noop = (): void => {};
 
   beforeAll(() => {
-    behaveLikeBaseSession.call(this, _buildInstance({ host: 'example.telnyx.com', login: 'login', password: 'password' }))
-    VertoHandler.call(this, Verto)
-    LayoutHandler.call(this, Verto)
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/tests/Verto.test.ts:23]: Removed these calls

File: packages/js/src/Modules/Verto/webrtc/VertoHandler.ts [hunk 3]
```diff
@@ -1,196 +1,240 @@
...
+        callOptions.telnyxCallControlId = params.telnyx_call_control_id;
+      }
+
+      if (params.telnyx_session_id) {
+        callOptions.telnyxSessionId = params.telnyx_session_id;
+      }
+
+      if (params.telnyx_leg_id) {
+        callOptions.telnyxLegId = params.telnyx_leg_id;
+      }
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/VertoHandler.ts:69]: Code to expose `telnyxSessionId` and other ids

File: packages/js/src/Modules/Verto/webrtc/interfaces.ts [hunk 4]
```diff
@@ -1,97 +1,101 @@
...
+  onNotification?: Function;
+  googleMaxBitrate?: number;
+  googleMinBitrate?: number;
+  googleStartBitrate?: number;
+  ringtoneFile?: string;
+  ringbackFile?: string;
+  // Optional, sent by Telnyx RTC
+  telnyxCallControlId?: string;
+  telnyxSessionId?: string;
+  telnyxLegId?: string;
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/interfaces.ts:36]: Added options call options from invite

**Review by SuaYoo — APPROVED** [review]

**Pull request merged**